ironic/drivers/modules/drac/raid.py

```diff
@@ -1064,8 +1103,12 @@ def create_configuration(self, task,
                 "logical disks to RAID mode for node %(node_uuid)s",
                 {'node_uuid': node.uuid})
 
+        #Check valid properties attached to volumes
+        if conversion_results:
+            logical_disks = _validate_volume_size(node, logical_disks_to_create)
+
         controllers = list()
-        for logical_disk in logical_disks_to_create:
+        for logical_disk in logical_disks:
```

Comment:
I don't think the change on the line above is correct.  If _validate_volume_size() is not called, then this change will cause it to create all of the virtual disks in logical_disks, which is incorrect.  I think the change on the line above should be reverted, and line 1108 should be changed to:
logical_disks_to_create = _validate...

Reply:
done